AdyenCard/Components/Card/CardComponentConfiguration.swift

```diff
@@ -20,6 +34,10 @@ extension CardComponent {
         /// Indicates whether to show the security code field at all. Defaults to true.
         public var showsSecurityCodeField: Bool
 
+        /// Indicates the display mode of the billing address form.
+        /// Defaults to none.
+        public var billingAddress: AddressFormType
```

Comment:
maybe not the best name for this variable. `billingAddressType` sounds better. right now it seems like the value of the property is some kind of model object

Reply:
ok, fair enough